ops/travis/travis-deploy.sh

```diff
@@ -23,8 +23,6 @@ with_python27() {
     bash -c "source $HOME/virtualenv/python2.7/bin/activate; $1"
 }
 
-should_deploy
```

Comment:
I still think this should be the first thing - we should maintain a way to killswitch the entire continuous pipeline if we need to